ACE/ace/config-g++-common.h

```diff
@@ -161,7 +161,9 @@
 // GCC >= 4.1 provides __sync_XXXX builtins for use in atomic operations
 // although the builtins are provided globally they are not supported on all platforms
 #if defined (ACE_HAS_THREADS)
-#if (__GNUC__ > 4) || ((__GNUC__ == 4) && (__GNUC_MINOR__ >= 1))
+#if (__GNUC__ > 4) || ((__GNUC__ == 4) && (__GNUC_MINOR__ >= 7))
```

Comment:
gcc 4.7 is the oldest GCC compiler we still support for ACE7, that has C++11 support, couldn't this part be simplified now?

Reply:
For master, yes, it sounds like you could just switch to the __atomics without need for checking compiler versions.